ui/lib/pki/addon/components/pki-sign-intermediate-form.hbs

```diff
@@ -0,0 +1,81 @@
+{{#if @model.id}}
+  {{! Model only has ID once form has been submitted and saved }}
+  <Toolbar />
+  <main data-test-issuer-details>
```

Comment:
I noticed the new page details component above, are lines 4 - 29 going to be pulled into the separate Page component?